draft-ietf-rats-corim.md

```diff
@@ -166,11 +166,19 @@ This specification describes the CoRIM format and documents how a Verifier shoul
 A Verifier needs to reconcile its various inputs, with CoRIM being one of them.
 In addition to the external CoRIM documents, the Verifier is expected to create an internal representation for each input and map each external representation to an internal one.
 By using the internal representation, the Verifier processes inputs as if they are part of a conversation, keeping track of who said what.
-The origin of the inputs is tracked as *authority*.
-The authority for the Claims in a CoRIM is the CoRIM issuer.
 
 Effectively, Attesters, Reference Value Providers, Endorsers, Verifier Owners, Relying Parties, and even the Verifier potentially all contribute to the conversation.
-Each producer of corresponding RATS Conceptual Messages can assert Claims about an Attester's actual or allowed state.
+These individual principals of the conversation may have their identity rooted back to some *authority* farther up the certificate chain than the signing key itself, such as a digital certificate authority intermediate or root key.
+The origin of contributions/inputs to the conversation are tracked at the granularity of authority, so CoRIM authors can predicate their reference measurements or endorsements based on existing properties rooted from the same authority.
+
+CoRIM inputs to the conversation include measurements within a specific Environment.
+A measurement never appears outside the scope of an associated Environment, so the Environment-Claim Tuple (ECT) is a data structure used to encapsulate all scope context for a given set of measurements.
+The authority for an ECT from a CoRIM is the CoRIM issuer.
+The collective content of an ECT may be referred to as a property, claim, or assertion when the content need not be presented in the ECT data structure.
```

Comment:
```suggestion
When the content of an a ECT data structure is presented to roles external to the Verifier, it MAY be referred to via different terms, such as property, claim, or assertion. Whenever an ECT is consumed by a role external to the Verifer, the consuming party MUST always attribute provenance and the ECT's intended use.
```

Reply:
What is "intended use"? I'm not sure how to comply with that. My code that touches it could be my intention because I'm a hot shot with "self-documenting code". Is that enough?

I thought ECT is a representation of an utterance in the conversation, with all scope-closing elements provided.